index.js

```diff
@@ -44,9 +44,10 @@ export function createStore(reducer) {
             roots.set(root, component);
             render();
         },
-        connect(component) {
+        connect(component, selector = state => state) {
```

Comment:
just a nit / question -- is your preference to keep the selector as the second arg, vs the react-redux style

```javascript
const ConnectedFoo = connect(state => ({ bar: state.foo.bar }))(Foo);
```

Reply:
I guess (to kinda answer my own question) your  way allows connecting to the state without any selectors.

Reply:
which would need to be something like the following in react-redux style,

```
const ConnectedFoo = connect()(Foo);
```